packages/gatsby/src/utils/start-server.ts

```diff
@@ -314,5 +329,12 @@ export async function startServer(
     socket?.to(`clients`).emit(`reload`)
   })
 
-  return { compiler, listener, webpackActivity, websocketManager, workerPool }
+  return {
+    compiler,
+    listener,
+    webpackActivity,
+    websocketManager,
+    workerPool,
+    webpackWatching: webpackDevMiddlewareInstance.context.watching,
```

Comment:
should this be a function that returns the value? I'm guessing that if webpackDevMiddlewareInstance changed this value, we wouldn't get an update on this returned object.

Reply:
No, we wouldn't - but that would only happen if we completely stop watching and start new one (different than suspending current watching and resuming it) and neither we nor middleware does that (at least now)

Reply:
As for making this a function - I don't really have anything against, so I'll go ahead and do that

Reply:
changed to `getWebpackWatching` getter